src/ListThumbnail/ListThumbnail.scss

```diff
@@ -38,6 +39,7 @@
   color: $text-color;
   width: 75%;
   overflow: scroll;
```

Comment:
We might want to try a couple different browsers, machines to see if the overflow: scroll is displaying correctly.